cmd/federation-controller/main.go

```diff
@@ -78,6 +102,15 @@ func parseFlags() {
 }
 
 func main() {
+	flag.StringVar(&metricsAddr, "metrics-bind-address", ":8080", "The address the metric endpoint binds to.")
+	flag.StringVar(&probeAddr, "health-probe-bind-address", ":8081", "The address the probe endpoint binds to.")
+	flag.BoolVar(&enableLeaderElection, "leader-elect", false,
+		"Enable leader election for controller manager. "+
+			"Enabling this will ensure there is only one active controller manager.")
```

Comment:
Why not adding it to `parseFlags` where we have other flags defined?

Reply:
Sorry, I was so focused on setting up the controller correctly that I totally forgot about that. Done

Reply:
I know :) and that's what reviews are for.